features/beforeunload.yml

```diff
@@ -1,9 +1,12 @@
 name: Beforeunload
 description: "The `beforeunload` event is fired when the current window is about to be unloaded. Typically this is used to display a dialog to confirm if users really want to leave the page when there is unsaved data that would be lost."
 spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#the-beforeunloadevent-interface
+status:
+  compute_from: api.Window.beforeunload_event
```

Comment:
This is the entry point, and aligns w/ the actual core feature availability better.